via/static/scripts/video_player/components/VideoPlayerApp.tsx

```diff
@@ -179,67 +229,40 @@ export default function VideoPlayerApp({
   return (
     <div
       data-testid="app-container"
-      className={classnames(
-        // Constrain height of interface to viewport height
-        'flex flex-col h-[100vh] min-h-0'
-      )}
+      className="flex flex-col h-[100vh] min-h-0"
     >
-      <div
-        data-testid="top-bar"
-        className={classnames(
-          // Match height of sidebar's top-bar
-          'h-[40px] min-h-[40px]',
-          'w-full flex items-center gap-x-3 px-2 border-b',
-          // Background is one shade darker than sidebar top-bar
-          'bg-grey-0'
-        )}
-      >
-        <a
-          href="https://web.hypothes.is"
-          target="_blank"
-          rel="noreferrer"
-          title="Hypothesis"
+      {multicolumn && (
+        <div
+          data-testid="top-bar"
+          className={classnames(
+            'h-[40px] min-h-[40px] w-full flex items-center gap-x-3',
```

Comment:
Remembering which of `justify-content: center` vs `align-items: center` applies to the main vs cross axis is bad enough. `items-center` is perhaps even worse.

Reply:
I precisely had some "fun" in the morning with this, trying to align some items on a flex container 😅